net/libfab/libfab.c

```diff
@@ -1474,10 +1230,11 @@ static int libfab_active_ep_create(struct m0_fab__ep *ep, struct m0_fab__tm *tm)
 	}
 
 	net = &ep->fep_nep;
+	net->nep_xprt_pvt = ep;
```

Comment:
Need to make this NULL or not needed?

Reply:
Not needed as `struct m0_net_end_point` is embedded into `struct m0_fab__ep` and when `struct m0_fab__ep` is freed, it will also free the sub structure.